### DISCUSSION TITLE
Welcome to pyqtgraph Discussions!

### DISCUSSION THREAD

**Discussion by j9ac9k** [pyqtgraph/pyqtgraph#1703] [General]
## 👋 Welcome!
  PyQtGraph maintainers are using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions about how to use or contribute PyQtGraph
  * Share ideas about PyQtGraph Development
  * Interact with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by smishy05** [comment]
Hey there @j9ac9k !
This is Shailesh here. I have been working on research space in the field of distributed systems and now, I am looking forward to contribute to open-source.
I had some queries regarding solving the issues. Suppose, I wish to resolve a issue that has been labelled as a ***bug***, then do I have to get it assigned to me by someone first or should I start working on a PR right away? Please let me know.

**Reply by j9ac9k** [reply]
Hi Shailesh!

We are not so formal as to track assignments, if there is an open issue and it appears it is inactive, have at it.  If you think the fix has other potential implications to the library and would like some guidance on how to proceed, by all means ask away.  

I sometimes assign issues to other maintainers if I think it's an area of the library they have more familiarity with.

Hope that helps, we look forward to seeing a PR from you!

**Reply by smishy05** [reply]
Thanks a lot for the reply! It did help a lot!

Sure! I will try to give my best and solve some issues!
